packages/ckeditor5-ckbox/docs/features/ckbox.md

```diff
@@ -46,66 +32,194 @@ Image files are inserted into the content as images that you can drag around and
+            </td>
+		</tr>
+    </tbody>
+</table>
+
+To find out more about CKBox, the brand-new file manager and image editor, visit the [CKBox website](https://ckeditor.com/ckbox/) and read the dedicated [CKBox documentation page](https://ckeditor.com/docs/ckbox/latest/guides/index.html). 
+
+You can read more about the storage options in the dedicated [CKBox Deployment](https://ckeditor.com/docs/cs/latest/onpremises/ckbox-onpremises/deployment.html) guide.
+
 ## Installation
 
-To use this feature in your application, you must first load the CKBox library and then enable CKBox integration in your rich-text editor instance.
+CKBox offers two primary methods for loading the library: using a Content Delivery Network (CDN) for fast, script-based integration, or using a Package Manager (npm/yarn) for modern application bundles.
 
-The easiest way to load the CKBox library is to include the `<script>` tag loading the `ckbox.js` file first:
+### Method 1: Installation from CDN
+
+The easiest way to load the CKBox library is by including a `<script>` tag in your HTML header that points to the CKBox CDN. You must include the script tag loading the ckbox.js file first:
 
 ```html
-<script src="https://cdn.ckbox.io/ckbox/2.4.0/ckbox.js"></script>
+<script src="https://cdn.ckbox.io/ckbox/{%CKBOX_VERSION%}/ckbox.js"></script>
 ```
 
-Please note, that while using the `latest` call instead of a version number is available, it is not advised. The latest version may bring breaking changes that will stall your CKBox integration.
-
-The CKBox feature requires one of the following plugins to be loaded to work correctly:
+When referencing the CKBox library via CDN, you must specify the version number (e.g., `{%CKBOX_VERSION%}` or `latest`).
 
-* {@link module:image/imageblock~ImageBlock The `ImageBlock` feature}
-* {@link module:image/imageinline~ImageInline The `ImageInline` feature}
-* {@link module:image/image~Image The `Image` feature} (a glue plugin that loads both the `ImageBlock` and `ImageInline` features)
+<info-box note>
+	While the ability to use the shorthand `latest` tag instead of a specific version number is available, it is strongly advised against. The use of the latest tag may inadvertently introduce breaking changes into your application, which could cause your CKBox integration to stall unexpectedly.
+</info-box>
 
-If you do not have any of these plugins in your editor, install one and add it to your plugin list.
+Suppose you wish to change the default language of the CKBox, which supports over 40 official translations. In that case, you must ensure you load the corresponding translation file after loading the main CKBox library. For example, to load the Spanish translation (es):
```

Comment:
Is there a list of available languages? If so, maybe it would be worth adding it here?